chatkit-core/src/main/kotlin/com/pusher/chatkit/util/DateUtil.kt

```diff
@@ -0,0 +1,13 @@
+package com.pusher.chatkit.util
+
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+internal val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.ENGLISH)
+
+internal object DateUtil {
```

Comment:
as per earlier feedback in Kotlin there shouldn't be a containing object, functions can be top-level

Reply:
👍 we discussed this in person -> have created a DateApiTypeMapper which has a `mapToEpochTime` method.